### ISSUE TITLE
Import Bitwarden codes from file

### ISSUE THREAD

**Issue by VioletRose** [beemdevelopment/Aegis#953]
I noticed today that Aegis doesn't support importing [Bitwarden](https://bitwarden.com) vault exports. These files may be .json or .csv, and currently don't have an option for file encryption, making this a comparatively simple inclusion.

For JSON, the relevant types (minus annotations) would be:
```java
class BitwardenItemLogin {
  /** May be an otpauth:// URI, or just a secret. */
  String totp;
}

class BitwardenItem {
  BitwardenItemLogin login;
  String name;
}

class BitwardenVault {
  List<BitwardenItem> items;
}
```

I am less familiar with deserializing .csv to POJOs, but the relevant column header is `login_totp`; in the example file I have, it is the 11th column.

TOTP support is currently a premium feature in Bitwarden, but I would be happy to help test the functionality; my personal device is rooted, so I should be able to install a test .apk if needed.

**Comment by alexbakker** [comment]
Thanks for the suggestion, but this is already implemented: #858. It's not included in a release yet though.

**Comment by VioletRose** [comment]
> Thanks for the suggestion, but this is already implemented: #858. It's not included in a release yet though.

Awesome, thank you. Suggestion for the future: maybe the issue for which the PR is completed shouldn't be closed until it's been in a release? When I was creating this issue, I did check for it in the open issues, but not closed because I assumed those would all be 100% done.

**Comment by alexbakker** [comment]
The GitHub issue tracker isn't really meant for that. It's a tool for developers and thus represents the current state of the master branch.
